docs/source/model-registry.rst

```diff
@@ -152,6 +152,57 @@ While the method above creates an empty registered model with no version associa
         run_id="d16076a3ec534311817565e6527539c0"
     )
 
+Run a MLflow Model from the Model Registry
+------------------------------------------
+
+After you have registered an MLflow model, you can run that model using :meth:`~mlflow.sklearn.load_model`. 
+
```

Comment:
Can we also mention `MlflowClient.get_model_version_download_uri(name, version)` as an explicit way to get the URI and download it? Might be also worth mentioning the java version. We don't need to provide an example for these I think because that distracts from the main flow - but we could link to the docs perhaps.

Reply:
I added a new section with an example.

Reply:
Perhaps we should add the param `artifact_uri` to `MlflowClient.downloadArtifacts`. It would be trivial to do so. What do you think?

Reply:
`MlflowClient.download_artifacts` is a run-specific API. Are you thinking we need an API to download any artifact URI? I think we used to have one but we cleaned it up as part of the 1.0 release. There is the internal `mlflow.tracking.artifact_utils._download_artifact_from_uri` (which i think eventually powers the CLI command below)... it would probably be useful to have a public method for that, but I think the community just hasn't asked for it (AFAIK).

Reply:
This section is tailored more toward the Model Registry Workflow API; we might not want to conflate it. It could serve a better purpose to make this part of the API calls in the `MLflowclient` docs.

Reply:
A couple of observations.

1. "Run an MLflow Model from the Model Registry"

This heading is not consistent with the use of gerunds in the other subheadings under this workflow. Also, the verb "Run", not sure it quite fits with model: you serve a mode; you train a model; you score (or infer from) a model. "Run" usage may be more appropriate with, say, something like "Run" an MLflow Project. 

I think more fitting to be consistent with the flow of the workflow APIs would be "Fetching (or Retrieving) an MLflow model" 
(We don't need to use the prepositional phrase "from the Model Registry,"  as we are already in the section). 

2. Second, you can fetch models from the registry using both native `load_model` and `pyfunc_load_model`. I think this is imperative because to serve a model from the CLI, we only support pyfunc flavors.

Reply:
ok looking above at the section intro, this is meant to be for API workflow using the python `MlflowClient` and `models` methods, so it makes sense not to include the model downloading stuff here. 

I still think it is very valuable for those who don't want to load the model using `<flavor>.load_model` -- could it make sense to include the downloading example as a whole different section for how to use model registry at inference time in arbitrary contexts? then we can list the python (`MlflowClient.get_model_version_download_uri`), REST API, and java ways. too much?

Reply:
Yes, true, we should evolve these docs, as part of the model registry general usage, with expanded sections: Accessing or downloading model artifacts: a) models b) gifs/pdfs c) schemas d) tags/values, etc.

Reply:
ok we can do this in another PR at a later date

Reply:
Yes, we can!